### DISCUSSION TITLE
Hoppscotch Desktop: Roadmap and Licensing

### DISCUSSION THREAD

**Discussion by cjchand** [hoppscotch/hoppscotch#3654] [Q&A] [answered]
Hey team!

First off, many thanks for all the hard work and the great community you've built here. Definitely keeping tabs on progress and wish you all the best!

Curious on any documented roadmap and licensing for Hoppscotch Desktop. 

I've got "rug pull licensing" PTSD from other projects, so don't want to start socializing the desktop app as a replacement for Postman's "scratchpad" capabilities without knowing any future plans to start charging per-seat, requiring cloud accounts like Postman did, etc.

Thanks in advance!

Cheers,
Chris

**Comment by cjchand** [comment]
Happy New Year, all! Any feedback on the above @AndrewBastin or @liyasthomas?

**Comment by liyasthomas** [comment, marked as answer]
Our desktop app is built on top of the [Hoppscotch Web Client](https://docs.hoppscotch.io/documentation/clients/web) hence we don't have a separate roadmap dedicated to the Hoppscotch desktop application.

The Hoppscotch desktop app's source code is licensed under MIT license and is available on our mono repo: https://github.com/hoppscotch/hoppscotch/tree/main/packages/hoppscotch-selfhost-desktop

**Comment by cjchand** [comment]
Sounds good and thanks for the quick response!
